third_party/2and3/chardet/langbulgarianmodel.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Tuple
+from . import _LangModelType
+
+Latin5_BulgarianCharToOrderMap: Tuple[int]
```

Comment:
Should be `Tuple[int, ...]` (an arbitrary-length tuple), not `Tuple[int]` (a tuple of length 1).

Technically this seems to be about a 255-tuple, but I don't really think we need a `Tuple[int, int, ... 255 times]`.

This also goes for the other `Tuple[int]`s in these stubs.

Reply:
I've changed it.